### DISCUSSION TITLE
Flanking Sequence Export Configuration

### DISCUSSION THREAD

**Discussion by jrodiger-lifemine** [GMOD/jbrowse-components#3212] [Q&A] [answered]
Currently it's possible to export a gene w/ 500bp up+down stream but is there a way to configure this to allow exporting a gene w/ 1kb up+down stream? This would be very useful for our team.

[https://jbrowse.org/jb2/docs/combined/#getting-the-protein-sequence-for-features](https://jbrowse.org/jb2/docs/combined/#getting-the-protein-sequence-for-features)

**Comment by cmdcolin** [comment, marked as answer]
this is definitely a good idea, it was actually implemented in https://github.com/GMOD/jbrowse-components/pull/2986 but needs to get landed maybe separate from the codon table thing
